### PR TITLE
JGZ conversies 1.0.0

### PR DESCRIPTION
**Pull request by ogieling**
https://bits.nictiz.nl/projects/JGZ/versions/13970

### CODE REVIEW

**Review by ahenket — APPROVED** [review]
I have checked the syntax, and generic flow aspects.

File: hl7_2_hl7/jeugdgezondheidszorg/dossieroverdracht-327-vs-400/dob-327-to-400.xsl [hunk 1]
```diff
@@ -1343,6 +1343,30 @@
         </xsl:choose>
     </xsl:template>
     
+    <xd:doc>
+        <xd:desc>Voor‑ of buitenschoolse voorzieningen/school: R008 - templateId 2.16.840.1.113883.2.4.6.10.100.108, ID @root vervangen voor nullFlavor NI</xd:desc>
+    </xd:doc>
+    <xsl:template match="hl7:patientPerson/hl7:asPatientOfOtherProvider/hl7:subjectOf/hl7:careProvision/hl7:performer/hl7:assignedProvider/hl7:id[not(@nullFlavor)] |
+                         hl7:patientPerson/hl7:asPatientOfOtherProvider/hl7:subjectOf/hl7:careProvision/hl7:performer/hl7:assignedProvider/hl7:representedOrganization/hl7:id[not(@nullFlavor)]" mode="dob327">
+        <xsl:copy>
```
Comment (pieter-edelman-nictiz) [inline hl7_2_hl7/jeugdgezondheidszorg/dossieroverdracht-327-vs-400/dob-327-to-400.xsl:1351]: I don't think this `<xsl:copy>` is really necessary?

**Review by pieter-edelman-nictiz — COMMENTED** [review]
Superficially, this looks fine to me.

File: hl7_2_hl7/jeugdgezondheidszorg/dossieroverdracht-327-vs-400/dob-327-to-400.xsl (hunk 1 above)
Comment (ogieling) [reply hl7_2_hl7/jeugdgezondheidszorg/dossieroverdracht-327-vs-400/dob-327-to-400.xsl:1318]: Without the `<xsl:copy>` the transformation fails.

**Pull request merged**